packages/terra-modal-manager/CHANGELOG.md

```diff
@@ -4,6 +4,10 @@ Changelog
 Unreleased
 ----------
 
+### Changed
+* Use DisclosureManager for state management and presentation.
+* Deprecate Redux APIs
```

Comment:
We should move this bullet to its on section:

```
### Deprecated 
* Deprecate Redux APIs
```

I've been using http://keepachangelog.com/en/1.0.0/ as a guide for our changelogs.